admiral/debug_functions.sqf

```diff
@@ -287,6 +287,88 @@ adm_debug_fnc_updateLineMarker = {
     DEBUG("admiral.debug",FMT_3("Updated line marker '%1' position between '%2' and '%3'.",_markerName,_posFrom,_posTo));
 };
 
+adm_debug_fnc_createDebugCounterMarker = {
+    FUN_ARGS_3(_side,_type,_xPos);
+    
+    PVT_2(_marker,_markerType);
+    _markerType = if (_type != "total") then {
+        _type;
+    } else {
+        COUNTER_DEBUG_MARKER_TOTAL;
+    };
+    _marker = [format ["adm_counter_%1_%2", _side,_type], [_xPos, 50, 0], "ICON", _markerType, [_side] call adm_debug_fnc_getSideColor, COUNTER_DEBUG_MARKER_SIZE] call adm_common_fnc_createLocalMarker;
+    _marker setMarkerTextLocal "0";
+    
+    _marker;
+};
+
+adm_debug_fnc_createDebugFactionCounterMarkers = {
+    FUN_ARGS_2(_side,_xPos);
+
+    DECLARE(_debugMarkers) = [];
+    PVT_1(_marker);
```

Comment:
If you have a at least 2 variables in a scope that need `private`, put them in the same `private []`.